course/models.py

```diff
@@ -402,7 +399,7 @@ class ParticipationRole(models.Model):
     identifier = models.CharField(
             max_length=100, blank=False, null=False,
             help_text=_("A symbolic name for this role, used in course code. "
-            "lower_case_with_underscores, no spaces. May be any string. The "
+            "Should be a valid identifier. The "
```

Comment:
```suggestion
            "Should be a valid identifier (as defined by Python). The "
```

Reply:
OK. I'll also add that to ptag's helptext.